lazy_relay.js

```diff
@@ -344,8 +399,10 @@ function onReadError(err) {
         );
     }
 
+    var conn = self.conn;
     self.onError(err);
-    self.conn.resetAll(err);
+
+    conn.resetAll(err);
```

Comment:
`LazyRelayInReq#onError` may `free()`, so we need to save a ref to `conn` in order to call `resetAll`.